src/wilson/dom.cljs

```diff
@@ -13,12 +13,11 @@
              [:th (capitalize k)]))]
     (into [:tbody]
           (for [row rows]
-            (into [:tr
-                   {:class (row->cls row)}]
+            (into [:tr (or (row->attrs row) {})]
                   (for [k keys]
                     [:td (get row k)]))))])
   ([keys rows]
-   (table keys rows {:row->cls :wilson/row-class})))
+   (table keys rows {:row->attrs {}})))
```

Comment:
Right. So `(constantly {})` solves that because it is a function that will always return `{}` when called no matter what; so you have a "real" default `row->attrs` fn that always returns something sane and you don't have to `(or ...)` hack around it.